app/views/admin/creator_settings/_form.html.erb

```diff
@@ -38,14 +38,18 @@
                          title: "Provide a valid HEX Color or pick your color from the color picker.",
                          placeholder: ::Settings::UserExperience.primary_brand_color_hex,
                          class: "crayons-textfield js-color-field",
-                         "data-action": "change->creator-settings#updateBranding" %>
+                         "data-action": "change->creator-settings#handleValidationsAndUpdates",
+                         "data-creator-settings-target": "brandColor" %>
       <%= color_field_tag :primary_brand_color_hex,
                           ::Settings::UserExperience.primary_brand_color_hex,
                           pattern: "^#+([a-fA-F0-9]{6})$",
                           placeholder: ::Settings::UserExperience.primary_brand_color_hex,
                           class: "crayons-color-selector js-color-field ml-2",
-                          "data-action": "change->creator-settings#updateBranding",
-                          required: true %>
+                          required: true,
+                          "data-action": "change->creator-settings#handleValidationsAndUpdates" %>
```

Comment:
A change on the brand color now needs to validate the color contrast and update the brand colors on the page. I've documented this all in the stimulus controller too :)